dietpi/dietpi-software

```diff
@@ -4569,14 +4569,14 @@ _EOF_
 
 			Banner_Installing
 
-			#Binary URL fails connection test, so we need to parent back a little: https://github.com/Fourdee/DietPi/issues/445#issuecomment-283400449
-			INSTALL_URL_ADDRESS='https://github.com/blynkkk/blynk-server/releases'
+			INSTALL_URL_ADDRESS='https://api.github.com/repos/blynkkk/blynk-server/releases/latest'
 			G_CHECK_URL "$INSTALL_URL_ADDRESS"
 
-			mkdir -p /etc/blynkserver
-
-			INSTALL_URL_ADDRESS='https://github.com/blynkkk/blynk-server/releases/download/v0.36.2/server-0.36.2-java8.jar'
-			wget "$INSTALL_URL_ADDRESS" -O /etc/blynkserver/server.jar
+            no_check_url=$(curl -s $INSTALL_URL_ADDRESS | grep -m1 'browser_download_url' | cut -d \" -f 4)	
+	
+            mkdir /usr/local/bin/blynk
+			wget "$no_check_url" -O /usr/local/bin/blynk/blynkserver.jar
+            chown -R dietpi:dietpi /usr/local/bin/blynk
```

Comment:
@joaofl 
Currently not sure, if there was a reason to leave blynk running as root user. But needs careful testing at least, before switching.

If switching user works, it should be not `dietpi`, but an own user `blynk` instead. `Group=dietpi` could be added additionally, in case software cross-access is required (blynk accessing other files within /mnt/dietpi_userdata, downloads, media or such). Never used blynk, so can't say something about that 😉. Blynk could/should then also be configured to create files with 77X/66X permissions, for incoming cross-access permissions (another software that needs to access blynk files).

Reply:
Dont think the server needs to interact at all with anything else on the system. So, for now, lef with the default permissions, being part of no group.